results-processor/processor.py

```diff
@@ -19,12 +20,13 @@
 import wptreport
 from wptscreenshot import WPTScreenshot
 
-
 _log = logging.getLogger(__name__)
 
 
 class Processor(object):
     USERNAME = '_processor'
+    # Timeout waiting for remote HTTP servers to respond
+    TIMEOUT_WAIT = 10
```

Comment:
Isn't this too short to fetch a zipfile full of screenshots over a slow network? Do you know what the actual times are in the processor before this change?

Or is the timeout only for the server to send *headers* and the response body can be arbitrarily slow after that?

Reply:
This is the timeout waiting for the server to start sending any byte. Unfortunately, I don't see an easy way to control the overall timeout provided by `requests`.

Reply:
Alright, good in this context. I guess if a server begins responding, then not continuing to send data will eventually cause the same type of error as if a network cable is cut, some TCP-level timeout.